### DISCUSSION TITLE
border class adding default color and not removing

### DISCUSSION THREAD

**Discussion by mabdullahkhalil** [tailwindlabs/tailwindcss#11691] [Help] [answered]
**What version of Tailwind CSS are you using?**

For example: v3.3.3

**What build tool (or framework if it abstracts the build tool) are you using?**

I am using Ruby on Rails, and npm package 14.19.0 to compile

**What version of Node.js are you using?**

For example: v14.19.1

**What browser are you using?**

For example: Chrome

**What operating system are you using?**

For example: macOS

**Describe your issue**

I have added `border` class to a component and upon adding `border-red-500` it is not changing the color of the border. However it works when I user `border-2`
I see that the default border class is set, can someone guide?

<img width="1441" alt="Screenshot 2023-07-26 at 6 32 48 PM" src="https://github.com/tailwindlabs/tailwindcss/assets/25610805/f0b5f627-5980-48c3-8742-925c4e394236">

**Comment by wongjn** [comment, marked as answer]
That border class does not seem like one that would be generated in a default Tailwind set up. Consider checking any custom CSS or Tailwind plugins for that rule and adjust to taste.
